src/main/resources/static/css/layout.css

```diff
@@ -107,12 +107,10 @@ html, body, .spinner {
     min-height: 200px;
 }
 
-
 /*file-specific CSS*/
 
 .checkbox_layout {
-    padding-top: 22px;
```

Comment:
WATCH OUT!!! you're changing a class used on another template. Removing these values fucks up the styling of the previous implemented features. Create your own class if necessary! Reinstate the previous values for this class please